test/harness/e2e/vm/cloudinit.go

```diff
@@ -0,0 +1,43 @@
+package vm
+
+import (
+	"errors"
+	"fmt"
+	"os"
+	"os/exec"
+	"path/filepath"
+)
+
+func (b *TestVM) ParseCloudInit() error {
```

Comment:
why "b" here? the rest of the receiver params are "v"s in other files. Just curious if it was a typo.